README.md

```diff
@@ -83,6 +85,41 @@ defmodule Scaling.Pipeline do
 end
 ```
 
+### `Membrane.FFmpeg.SWScale.PixelFormatConverter`
+
+Converting pixel format of an encoded (using H.264 standard) video requires a parser and a decoder
+because the Converter performs convertion only on the raw video. The pipeline changes the pixel
+format of the video to the `I422` format.
```

Comment:
```suggestion
Converting the pixel format of an encoded (using H.264 standard) video requires a parser and a decoder
because the Converter performs conversion only on the raw, decoded video. The pipeline changes the pixel
format of the video to the `I422` format and reencodes it.
```